projects/cega/evm.js

```diff
@@ -0,0 +1,100 @@
+
+const sdk = require('@defillama/sdk')
+const abi = require("./abi.json");
+
+const usdcDecimals = 10 ** 6;
+const maxLeverage = 5;
+const CEGA_PRODUCT_VIEWER = "0x31C73c07Dbd8d026684950b17dD6131eA9BAf2C4";
+const FCN_PRODUCTS = [
```

Comment:
Can we read product names from CegaState contract instead? Otherwise Defillamma will be stale with each new product release / LOV update?

Reply:
yup done!